rocky/rocky/views/ooi_view.py

```diff
@@ -69,6 +69,23 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class OOICreateView(SingleOOIMixin):
```

Comment:
What's the point of introducing another extra (abstract) class if it's only inherited by one child? I don't see how this refactoring step improved the previous situation

Reply:
I have moved the object creation out of the class, because the BaseOOIDetailView have many other initiation stuff that I don't need. There are many things there that I don't need.

Reply:
It is also inherited at MuteFindingsBulkView and BaseOOIFormView

Reply:
Maybe I can move the function into SIngleOOIMixin itself

Reply:
@ammar92 Moved the function to the SingleOOIMixin